core/templates/dev/head/pages/admin/admin.mainpage.html

```diff
@@ -70,6 +70,23 @@
 
     {% include 'pages/footer_js_libs.html' %}
 
+    <script src="/templates/dev/head/pages/admin/admin.module.js"></script>
+    <script src="/templates/dev/head/pages/admin/admin.controller.js"></script>
+    
+    <script src="/templates/dev/head/pages/admin/config_tab/admin-config-tab.module.js"></script>
+    <script src="/templates/dev/head/pages/admin/roles_tab/admin-roles-tab.module.js"></script>
+    <script src="/templates/dev/head/pages/admin/roles_tab/roles-graph/role-graph.module.js"></script>
+    <script src="/templates/dev/head/pages/admin/admin-navbar/admin-navbar.module.js"></script>
+    <script src="/templates/dev/head/pages/admin/misc_tab/admin-misc-tab.module.js"></script>
+    <script src="/templates/dev/head/pages/admin/jobs_tab/admin-jobs-tab.module.js"></script>
+
```

Comment:
Don't have newlines in between the script tags of the same page. This would cause confusion with the grouping convention of different page scripts.

Reply:
done